source/default/_patterns/02-molecules/accordion/_accordion.twig

```diff
@@ -0,0 +1,49 @@
+{#
+/**
+ * @file
+ * accordion
+ *
+ * Available variables for accordion:
+ *  - accordion.data [array]: array of tab objects.
+ *    - title [string]: Accordion title.
+ *    - panel [string]: Panel content.
+ *  - accordion.collapsed [bool]: Will start will all panels collapsed.
+ *  - accordion.allow_toggle [bool]: Allow for each toggle to both open and close its section. Makes it possible for all sections to be closed. Assumes only one section may be open.
+ *  - accordion.allow_multiple [bool]: Allow for multiple accordion sections to be expanded at the same time. Assumes data-allow-toggle otherwise the toggle on open sections would be disabled.
+ */
+#}
+{% set seed = random() %}
+{% set trigger_element = accordion.trigger_element|default('h3') %}
+<div class="accordion mb-4 focus-within:shadow-outline"
+  {{ accordion.allow_toggle or accordion.collapsed ? 'data-allow-toggle="true"' }}
+  {{ accordion.allow_multiple ? 'data-allow-multiple="true"' }}
+>
+  {% for item in accordion.data %}
+    {% set item_uid = "#{seed}-#{loop.index}" %}
+    <{{ trigger_element }}>
+      <button {{ loop.first and not accordion.collapsed ? 'aria-expanded="true"' }}
```

Comment:
Can I talk you into using an attributes array and the `attributify` filter? We could use some better examples of it working in Particle and it would also allow us to update any amount of attributes in markup.